docs/targets/solana.rst

```diff
@@ -191,10 +191,10 @@ or a hex string with the format ``hex"4142"``, or a constructor argument of type
 
 .. _value_transfer:
 
-Transfering native value with a function call
+Transferring native value with a function call
 _____________________________________________
```

Comment:
The title is now longer so you need an extra `_` to make the tests pass, thanks

Reply:
Sure! Just added it